docs/CHANGELOG.asciidoc

```diff
@@ -30,6 +30,223 @@ This section summarizes the changes in each release.
 
 coming[7.3.0]
 
+See also <<breaking-changes-7.3, breaking changes in 7.3>>.
+
+[float]
+[[enhancement-7.3.0]]
+=== Enhancements
+Canvas::
+* Feat: Keyboard shortcuts for nudging elements {pull}39208[#39208]
+* [Fixes #27123]  Feat: Zoom In/Out {pull}38832[#38832]
+* Add joinRows Canvas expression function {pull}38680[#38680]
+* [Fixes #27124]  Feat: Fit Workpad to Window {pull}39864[#39864]
+* [Fixes #23061, #23144, #29526] Enable Canvas URL Parameters {pull}40061[#40061]
+* Alignment and distribution {pull}39132[#39132]
+* [Fixes #30841]  Feat: Box select {pull}32995[#32995]
+Code::
+* Change index name generation rules for code {pull}40959[#40959]
+* Always read file content from the original bare repo in indexer {pull}40176[#40176]
+* Add ctags langserver as a dependency and launch it {pull}39763[#39763]
+* Remove ctags langserver from developing list {pull}40258[#40258]
+* Don't use symlink for jdt config on windows {pull}40338[#40338]
+* Add ctags icon {pull}40263[#40263]
+* Api for lsp initialize state {pull}39425[#39425]
+* Adjust oom score, make linux OOM killer kill subprocesses first  {pull}39499[#39499]
+* Parse and register language file extension from monaco-editor {pull}40578[#40578]
+* Use status api to decide whether a lsp request should be sent {pull}41310[#41310]
+Geo::
+* [Fixes #34399] [Maps] add Where clause to terms joins {pull}39593[#39593]
+* [Maps] increase document limit from 2048 to 10000 {pull}39697[#39697]
+* [Maps] remove beta labeling from application and docs {pull}39871[#39871]
+* [Fixes #31697] [Maps] Custom labels for metric aggregations {pull}39914[#39914]
+* [Maps] replace legacy map visualizations with maps in sample data dashboards {pull}35997[#35997]
+* [Fixes #34026] [Maps] make EMS tooltips configurable {pull}34325[#34325]
+* [Fixes #29005] [Maps] add multiple feature selection in tooltips {pull}39629[#39629]
```

Comment:
Let's remove this section to "Maps" (which is what the app is called in the UI) and remove [Maps] prefixes since they will be redundant ..